### PR TITLE
Zord sheet redesign

### PR DESCRIPTION
**Pull request by guswadsworth12**
Large changes to the Zord and Megazord sheet. Also edited some UI elements.

### CODE REVIEW

File: lang/en.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     "TypeGear": "Gear",
     "TypeWeapon": "Weapon",
     "TypeWeaponupgrade": "Weapon Upgrade",
-    "TypeZordcombiner": "Zord Combiner",
+    "TypeZordcombiner": "Zord Component",
```
Comment (phildominguez) [inline lang/en.json:16]: I think we should leave these as Zord Combiner until we change it everywhere at the same time.

File: css/essence20.css [hunk 2]
```diff
@@ -1385,4 +1385,19 @@ form .notes, form .hint {
   display: flex;
   flex-direction: row;
   justify-content: flex-start;
+}
+
+/* ----- ZORD SHEET STYLING ---- */
```
Comment (phildominguez) [inline css/essence20.css:1381]: I assume we can split these up into separate files somehow, yeah? I can't handle 1400 lines lol. Can put this off until next change, though.

File: templates/actor/parts/actor-zordcombiner.hbs [hunk 3]
```diff
@@ -0,0 +1,20 @@
+<span class="flex-group-center">
```
Comment (phildominguez) [inline templates/actor/parts/actor-zordcombiner.hbs:1]: 4 space indentations in this file 😲

**Review by phildominguez — CHANGES_REQUESTED** [review]
👍 Lmk if you need assistance with the conflicts 👍

**Comment by phildominguez** [comment]
> Lots of conflicts here. Holy moly.

LOL yeah, it happens. Generally it's a good idea to keep PRs smaller and focused on one feature/fix to avoid conflicts.

File: css/essence20.css (hunk 2 above)
Comment (guswadsworth12) [reply css/essence20.css:1381]: This will involve getting ourselves both set up with sass and less as a preprocessor. Will research.

**Review by phildominguez — APPROVED** [review]

**Pull request merged**